dashboard/app/models/levels/level.rb

```diff
@@ -733,6 +733,12 @@ def clone_with_suffix(new_suffix, editor_experiment: nil)
     end
 
     level.update!(update_params)
+
+    # Copy the level_concept_difficulty of the parent level to the new level
+    new_lcd = level_concept_difficulty.dup
+    level.level_concept_difficulty = new_lcd
+    level.save! if level.changed?
```

Comment:
I'd recommend we implement this in a way that matches the pattern established by the rest of the method.

```suggestion
    # Copy the level_concept_difficulty of the parent level to the new level
    update_params[:level_concept_difficulty] = level_concept_difficulty.dup
    
    level.update!(update_params)
```